### DISCUSSION TITLE
parseAsIsoDate would be nice

### DISCUSSION THREAD

**Discussion by tylersayshi** [47ng/nuqs#705] [Ideas] [answered]
Having just a plain date for when the timestamp is not necessary would be nice.

I'd be happy to help add this if it sounds good.

**Comment by franky47** [comment]
That would be a great addition to the community parsers!

Just need to find a way to organise those.

**Reply by franky47** [reply]
Actually scratch that, it would be useful in core, feel free to add it to parsers.ts (and add a demo in the docs). Thanks!

**Comment by tylersayshi** [comment]
Added with #707 ✅
